Branch-SDK/Branch-SDK/BNCDeviceInfo.m

```diff
@@ -124,7 +125,7 @@ + (NSString*) userAgentString {
             BNCPreferenceHelper *preferences = [BNCPreferenceHelper preferenceHelper];
             preferences.browserUserAgentString = browserUserAgentString;
             preferences.lastSystemBuildVersion = self.systemBuildVersion;
-			//NSLog(@"[Branch] userAgentString: '%@'.", browserUserAgentString);
+			BNCLogDebug(@"userAgentString: '%@'.", browserUserAgentString);
```

Comment:
@E-B-Smith  I don't fully understand the `BNCLog` code but my understanding is that methods like `BNCLogDebug` and `BNCLogWarning` don't check for flags like `isDebug` or `suppressLogWarnings`. Is this correct? If so, does this line mean that this log message will show up in all scenarios even in production apps that don't have `setDebug` enabled?